db/schema.rb

```diff
@@ -66,6 +66,7 @@
     t.uuid "bulk_assignment_id"
     t.uuid "visitor_supersession_id"
     t.string "context"
+    t.boolean "force", default: false, null: false
```

Comment:
These `null: false` values are hand-edited, so that future users will get the good version, and current users will get the safe version.

We'll have to be careful that subsequent PRs from people who had an existing local DB that did the migration version do not accidentally remote this constraint.